### ISSUE TITLE
Is there a pretrained NER model?

### ISSUE THREAD

**Issue by erip** [monologg/KoBERT-NER#1]
Thanks for this repo, @monologg! I'm interested in playing with it, but my Korean isn't so good - sorry if you've answered this already in your README. 😄 

I am trying to run NER using the below:

```python
>>> from transformers  import pipeline
>>> from kobert_transformers import get_tokenizer
>>> tokenizer = get_tokenizer()
>>> ner = pipeline("ner", "monologg/kobert", tokenizer=tokenizer)
>>> s = """이 대회 결승전은 축구 자체보다 125,000명 정도 되는 웸블리 스타디움의 수용인원을 훌쩍 뛰어넘 은 엄청난 인파로 인한 큰 혼란 때문에 더 중요하게 여겨진다. 추정 300,000여 명의 관중들이 입장하면서 경기장의 테라스까지 꽉 차서 사람들이 넘칠 정도였고, 관중들은 선수들이 경기하는 경기장 안 피치까지 내려가기 까지 했다. 말에 올라탄 경찰들이 경기 시작을 위해 피치 위의 관중들을 올려 보내려고 투입되었는데, 이때 경찰이 탄 말 중 밝은 색의 말이 이 날의 상징이 되었다."""
>>> for e in ner(s):
...     print(e)
...
{'word': '[CLS]', 'score': 0.5701352953910828, 'entity': 'LABEL_0'}
{'word': '이', 'score': 0.5712112188339233, 'entity': 'LABEL_0'}
{'word': '대회', 'score': 0.5417128801345825, 'entity': 'LABEL_0'}
{'word': '결승', 'score': 0.5183829665184021, 'entity': 'LABEL_1'}
{'word': '전', 'score': 0.5882442593574524, 'entity': 'LABEL_0'}
{'word': '은', 'score': 0.5698714256286621, 'entity': 'LABEL_0'}
{'word': '축구', 'score': 0.5152527689933777, 'entity': 'LABEL_0'}
{'word': '자체', 'score': 0.6081604957580566, 'entity': 'LABEL_0'}
{'word': '보다', 'score': 0.6378726959228516, 'entity': 'LABEL_0'}
{'word': '12', 'score': 0.5943517684936523, 'entity': 'LABEL_0'}
{'word': '5,000', 'score': 0.5709324479103088, 'entity': 'LABEL_0'}
{'word': '명', 'score': 0.5489593744277954, 'entity': 'LABEL_0'}
{'word': '정도', 'score': 0.61537104845047, 'entity': 'LABEL_0'}
{'word': '되는', 'score': 0.5776959657669067, 'entity': 'LABEL_0'}
{'word': '', 'score': 0.5248542428016663, 'entity': 'LABEL_0'}
{'word': '[UNK]', 'score': 0.6252432465553284, 'entity': 'LABEL_0'}
{'word': '블', 'score': 0.5442931652069092, 'entity': 'LABEL_1'}
{'word': '리', 'score': 0.5006702542304993, 'entity': 'LABEL_1'}
{'word': '스타', 'score': 0.5333179831504822, 'entity': 'LABEL_0'}
{'word': '디', 'score': 0.5351016521453857, 'entity': 'LABEL_0'}
{'word': '움', 'score': 0.6003034710884094, 'entity': 'LABEL_0'}
{'word': '의', 'score': 0.5910236835479736, 'entity': 'LABEL_0'}
{'word': '수용', 'score': 0.5944420695304871, 'entity': 'LABEL_0'}
{'word': '인', 'score': 0.5833531022071838, 'entity': 'LABEL_0'}
{'word': '원을', 'score': 0.5756275057792664, 'entity': 'LABEL_0'}
{'word': '', 'score': 0.5720658302307129, 'entity': 'LABEL_0'}
{'word': '훌', 'score': 0.5684361457824707, 'entity': 'LABEL_0'}
{'word': '쩍', 'score': 0.5795609951019287, 'entity': 'LABEL_0'}
{'word': '뛰어넘', 'score': 0.6168298721313477, 'entity': 'LABEL_0'}
{'word': '은', 'score': 0.5739412307739258, 'entity': 'LABEL_0'}
{'word': '엄청난', 'score': 0.5293063521385193, 'entity': 'LABEL_0'}
{'word': '인', 'score': 0.5658405423164368, 'entity': 'LABEL_0'}
{'word': '파', 'score': 0.5560188293457031, 'entity': 'LABEL_0'}
# ...
```

It looks like the code above isn't actually performing NER since the labels don't match those in your README.

It seems like I can train the NER model using your script and the data in the repo, but is there a pretrained model already? I couldn't seem to find one.

**Comment by monologg** [comment]
Hi:)

Sadly, I didn't uploaded the trained NER model. 

`monologg/kobert` is a naive pretrained weight like `bert-base-uncased` in English.

And I didn't take account of `pipeline`, so it gives `LABEL_0` that I didn't use in KoBERT-NER.

------

```bash
$ python3 main.py --model_type kobert --do_train
```

As written on the README, if you train with the command above, `model` directory will be created. 

With that checkpoint. you could get the better result:)

**Comment by erip** [comment]
Awesome! I'll give it a try. :smile: do you have a config for the results in your README; hyperparams, etc.?

**Comment by monologg** [comment]
Hyperparams are written on `main.py`. 

Also list of labels is in `data/label.txt` :)

**Comment by erip** [comment]
Awesome - so the default values in will yield the results in the table?

**Comment by monologg** [comment]
Almost similar with the table. You may change the `--seed` option to get a better result.

**Comment by monologg** [comment]
@erip 
I've made a new commit for writing the prediction during evaluation.

When you run the code, plz give `--write_pred` option.

The prediction will be written as the order of [`word`, `true label`, `pred_label`]

**Comment by erip** [comment]
Thanks so much! Training the model now. 😄
